hardware/raspberrypi/schematics/README.md

```diff
@@ -30,6 +30,10 @@ Schematics for the various Raspberry Pi board versions:
 
 - [Revision 1.1](rpi_SCH_1aplus_1p1_reduced.pdf)
 
+## Raspberry Pi Model B/A
+
+- [Revision 1.0](Raspberry-Pi-Schematics-R1.0.pdf)
```

Comment:
I left the filename as it was in the original source, even though it doesn't match all the others, since I figured that makes its provenance clearer.